src/app/modules/time-entries/pages/time-entries.component.ts

```diff
@@ -107,7 +107,7 @@ export class TimeEntriesComponent implements OnInit, OnDestroy {
       const isEditingEntryEqualToActiveEntry = this.entryId === this.activeTimeEntry.id;
       const isStartDateGreaterThanActiveEntry = startDateAsLocalDate > activeEntryAsLocalDate;
       const isEndDateGreaterThanActiveEntry = endDateAsLocalDate > activeEntryAsLocalDate;
-      if(!isEditingEntryEqualToActiveEntry && (isStartDateGreaterThanActiveEntry || isEndDateGreaterThanActiveEntry)){
+      if (!isEditingEntryEqualToActiveEntry && (isStartDateGreaterThanActiveEntry || isEndDateGreaterThanActiveEntry)) {
```

Comment:
`(isStartDateGreaterThanActiveEntry || isEndDateGreaterThanActiveEntry)` could be subtracted into a new const. 

`isTimeEntryOverlapping = isStartDateGreaterThanActiveEntry || isEndDateGreaterThanActiveEntry` then the if clause could only have this condition:

`if (!isEditingEntryEqualToActiveEntry && isTimeEntryOverlapping)`

Reply:
I only added space between them, but actually is a good opportunity to refactor that code.